src/assets/scripts/client/aircraft/FlightManagementSystem/Fms.js

```diff
@@ -804,7 +805,7 @@ export default class Fms {
      * @return {array<boolean, string>}
      */
     replaceArrivalProcedure(routeString) {
-        const routeStringElements = routeString.split(PROCEDURE_OR_AIRWAY_SEGMENT_DIVIDER);
+        const routeStringElements = routeString.toUpperCase().split(PROCEDURE_OR_AIRWAY_SEGMENT_DIVIDER);
```

Comment:
please remove this `.toUpperCase()` here.  have have, and continue to try, to normalize casing as `lowercase`.  even so, casing shouldn't matter here if you are doing a split.

Reply:
done

Reply:
@n8rzz actually, this one is needed. They are stored as uppercase in the routemodel.

Reply:
I know you want to get this in soon, could you add a comment to these as a reminder to do this in a less-dirty way?  I feel like this can be solved in a better way